test/OpenIddict.Server.AspNetCore.IntegrationTests/OpenIddictServerAspNetCoreIntegrationTests.cs

```diff
@@ -29,6 +30,11 @@ namespace OpenIddict.Server.AspNetCore.FunctionalTests
 {
     public partial class OpenIddictServerAspNetCoreIntegrationTests : OpenIddictServerIntegrationTests
     {
+        public OpenIddictServerAspNetCoreIntegrationTests(Xunit.Abstractions.ITestOutputHelper outputHelper)
```

Comment:
Can we add a `using` here or is the name ambiguous?

Reply:
Sure - I went with this one to reduce the diff size, but I'll add it.